arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -372,56 +372,33 @@ bool iresearchAnalyzerLegacyAnalyzers( // upgrade task
     }
   }
 
-  // register the text analyzers with the current vocbase
+  // create collection _analyzers
   {
-    // NOTE: ArangoDB strings coming from JavaScript user input are UTF-8 encoded
-    static const std::vector<std::pair<irs::string_ref, irs::string_ref>> legacyAnalzyers = {
-      { "text_de", "{ \"locale\": \"de.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_en", "{ \"locale\": \"en.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_es", "{ \"locale\": \"es.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_fi", "{ \"locale\": \"fi.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_fr", "{ \"locale\": \"fr.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_it", "{ \"locale\": \"it.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_nl", "{ \"locale\": \"nl.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_no", "{ \"locale\": \"no.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_pt", "{ \"locale\": \"pt.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_ru", "{ \"locale\": \"ru.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_sv", "{ \"locale\": \"sv.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-      { "text_zh", "{ \"locale\": \"zh.UTF-8\", \"ignored_words\": [ ] }" }, // empty stop word list
-    };
-    static const irs::flags legacyAnalyzerFeatures = { // add norms + frequency/position for by_phrase
-      irs::frequency::type(), // frequency feature
-      irs::norm::type(), // norm feature
-      irs::position::type(), // position feature
-    };
-    static const irs::string_ref legacyAnalyzerType("text");
-    bool success = true;
-
-    // register each legacy static analyzer with the current vocbase
-    for (auto& entry: legacyAnalzyers) {
-      auto name = normalizedAnalyzerName(vocbase.name(), entry.first);
-      auto& type = legacyAnalyzerType;
-      auto& properties = entry.second;
-      arangodb::iresearch::IResearchAnalyzerFeature::EmplaceResult result;
-      auto res = analyzers->emplace( // add analyzer
-        result, name, type, properties, legacyAnalyzerFeatures // args
+    if (!getAnalyzerCollection(vocbase)) {
+      static auto const properties = // analyzer collection properties
+        arangodb::velocypack::Parser::fromJson("{ \"isSystem\": true }");
```

Comment:
I think the collection have a `replicationFactor` attribute set as all the other system collections, see `createSystemCollection` in UpgradeTasks.cpp.
Otherwise the entire collection will become unavailable in a cluster if the one node goes down that has that collection's data.

Reply:
done